specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Batch Document Translation API",
+    "version": "v1.0-preview.1"
+  },
+  "host": "westus2.cognitiveservices.azure.com",
+  "basePath": "/translator/text/document",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/v1.0-preview.1/batches": {
+      "post": {
+        "tags": [
+          "Translation"
+        ],
+        "summary": "Submit a batch document translation request to the translation service",
+        "description": "Submit a batch request to the document translation service.\r\n            \r\nEach request can consists of multiple inputs.\r\nEach input will contains both a source and destination container for source and target language pair.\r\n            \r\nThe prefix and suffix filter (if supplied) will be used to filter the folders.\r\nThe prefix will be applied to the subpath after the container name\r\n            \r\nGlossaries / Translation memory can be supplied and will be applied when the document is being translated.\r\nIf the glossary is invalid or unreachable during translation time.  An error will be indicated in the document status.\r\n            \r\nIf the file with the same name already exists in the destination, it will be overwritten.\r\nTargetUrl for each target language needs to be unique.",
+        "operationId": "Batch Documents Submit",
```

Comment:
operationId should be "GroupName_OperationName", this will be reflected in SDK and document generation